requirements.txt

```
@@ -27,6 +27,7 @@ gql[requests]
 gspread==3.6.0
 gunicorn==20.0.4
 inflect==4.1.0
+ipaddress==1.0.*
```

@dbaty est-ce qu'il vaut pas mieux laisser la version libre ?

---

Oui, je suis plutôt d'avis de ne pas préciser la version. Il est assez improbable qu'il y ait une version 2, et qu'elle soit incompatible pour notre usage (très limité)... et si jamais c'est le cas, les tests sont là pour ça. :)

```suggestion
ipaddress
```